service/controller/key/key.go

```diff
@@ -41,9 +41,9 @@ const (
 	// TimeoutSeconds is TimeoutSeconds param in liveness probe config.
 	TimeoutSeconds = 5
 	// PeriodSeconds is PeriodSeconds param in liveness probe config.
-	PeriodSeconds = 35
+	PeriodSeconds = 45
```

Comment:
It takes 8 minutes for the master to start so I'm increasing this to avoid a restart loop.